docs/Bread.md

```diff
@@ -0,0 +1,89 @@
+## What is BREAD?
```

Comment:
Seems weird to have docs for this contract but to not include the contract. Will bring it up at next core meeting.